client/src/App.jsx

```
@@ -1,15 +1,43 @@
+      }
+      setIsInitialized(true);
+    };
+    initializeAuth();
+  }, []);
+
+  if (!isInitialized) return null;
+
   return (
     <>
       <Provider store={store}>
         <RouterProvider router={router}></RouterProvider>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)